src/polyfills/get-bounding-client-rect.js

```diff
@@ -18,7 +18,7 @@ let nativeClientRect;
 /**
  * Polyfill for Node.getBoundingClientRect API.
  * @this {!Element}
- * @return {!ClientRect|!LayoutRectDef}
+ * @return {!ClientRect|LayoutRectDef}
```

Comment:
```suggestion
 * @return {ClientRect|LayoutRectDef}
```

Reply:
Sure, but there are still tons of these exclamation points in this file. It isn't in Core.